src/components/updatemenus/attributes.js

```diff
@@ -57,6 +57,37 @@ module.exports = {
         ].join(' ')
     },
 
+    openreverse: {
```

Comment:
+1 for `direction`
I like the explicitness of `'l'|'r'|'u'|'d'` over `true|false`